e2e/init.js

```diff
@@ -0,0 +1,10 @@
+const detox = require('detox');
```

Comment:
Do we use that init file or one inside `__e2e__` folder?

Reply:
see above